cli/command/task_command_test.go

```diff
@@ -190,6 +190,12 @@ func TestGetTask(t *testing.T) {
 		Return([]string{"id"}, nil)
 
 	tc.Client.EXPECT().
+		ListTasks().
+		Return([]*models.TaskSummary{
```

Comment:
this looks gross and makes this test harder to read. Can you please split out the return value into its own variable?